superwise/metadata.csv

```diff
@@ -0,0 +1,5 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name
+superwise.metric.overall.data_drift,gauge,,day,,data drift,0,superwise,
+superwise.metric.overall.weighted_data_drift,gauge,,day,,weighted data drift,0,superwise,
+superwise.metric.overall.quantity,gauge,,day,,quantity,0,superwise,
+
```

Comment:
Are these all of the supported metrics? The readme mentions activity, incident, and model metrics.

Reply:
These are the basic metrics.
The Superwise incident is not a metric, but it is equivalent to an incident in Datadog.

Reply:
Sounds good! It's recommenced to list all known metrics here if possible

Reply:
To start, these are the metrics (: